src/shared/netif-naming-scheme.h

```diff
@@ -39,6 +39,7 @@ typedef enum NamingSchemeFlags {
         NAMING_BRIDGE_MULTIFUNCTION_SLOT = 1 << 14, /* Use PCI hotplug slot information associated with bridge, but only if PCI device is multifunction */
         NAMING_DEVICETREE_ALIASES        = 1 << 15, /* Generate names from devicetree aliases */
         NAMING_USB_HOST                  = 1 << 16, /* Generate names for usb host */
+        NAMING_SR_IOV_R                  = 1 << 17, /* Use "r" suffix for SR-IOV VF representors */
```

Comment:
SR_IOV_R -> SR_IOV_REPRESENTOR is better?

Reply:
I have just followed the name for virtual devices...
NAMING_SR_IOV_V for VFs
NAMING_SR_IOV_R for VF representors